UnityProject/Assets/Scripts/Health/Living/PlayerHealthReporting.cs

```diff
@@ -67,7 +69,11 @@ private void ServerMonitorHealth()
         {
             //Add other damage methods here like burning, 
             //suffication, etc
-
+            
+            //If already dead then do not check the status of the body anymore
+            if (playerMove.isGhost)
```

Comment:
No if's without { }! It's easy to miss, prone to produce errors and makes code harder to read!